charts/testkube-api/templates/_helpers.tpl

```diff
@@ -93,6 +93,53 @@ Define API image
 {{- end -}}
 {{- end -}}
 
+{{/*
+Define Test Workflows Toolkit Image
+*/}}
+{{- define "testkube-tw-toolkit.image" -}}
+{{- $registryName := .Values.imageWorkflows.registry -}}
+{{- $repositoryName := .Values.imageWorkflows.repositoryToolkit -}}
+{{- $tag := default .Chart.AppVersion .Values.imageWorkflows.tag | toString -}}
```

Comment:
`tag` should default to the API Server's tag when it's not available.

Reply:
What would be the other options?

Reply:
Done